Project/Assets/ML-Agents/Examples/Bouncer/Demos/ExpertBouncer.demo.meta

```diff
@@ -4,7 +4,7 @@ ScriptedImporter:
   fileIDToRecycleName:
     11400000: Assets/ML-Agents/Examples/Bouncer/Demos/ExpertBouncer.demo
   externalObjects: {}
-  userData: ' (MLAgents.Demonstrations.Demonstration)'
+  userData: ' (MLAgents.Demonstrations.DemonstrationSummary)'
```

Comment:
Does this mean that old demo files will be incompatible (nothing wrong with that, just making sure)

Reply:
No, shouldn't have any effect.